commit 66ed29a9ecd8d88c5172ea58baf69000aa8e0508
Author: Guenter Roeck <linux@roeck-us.net>
Date:   Wed Feb 8 14:02:59 2017 -0800

    hwmon: (it87) Ensure that pwm control cache is current before updating values
    
    commit 82dbe987b70042b340f851bdc969a971081e5f02 upstream.
    
    If sensor attributes were never read, the pwm control data has not been
    initiialized, which can cause wrong driver behavior. Ensure that cached
    data is current before acting on it.
    
    Reported-by: Kevin Folz <kfolz@evertz.com>
    Signed-off-by: Guenter Roeck <linux@roeck-us.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1240_66ed29a9ecd8d88c5172ea58baf69000aa8e0508.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1240_66ed29a9ecd8d88c5172ea58baf69000aa8e0508.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
